### ISSUE TITLE
Error "GPT PMBR size mismatch" after dd-ing the generated image

### ISSUE THREAD

**Issue by nikolas-n** [dimkr/devsus#3]
Hi there! 

After I have run .devsu.sh and generated `devuan-jessie-c201-libre-2GB.img` on a Debian Jessie machine, I `dd` it to a SD card but when I issue `cgpw show /dev/mmcblk1` I get the error `GPT PMBR size mismatch` and I cannot boot the chromebook from it. I also run `cgpt repair /dev/mmcblk1` but it's still not working. What could be the problem.

Also, would it be possible to have the img files as a torrent so that we know for sure that they're compiled right? That'd be great...

Anyway, this repo is awesome!

**Comment by dimkr** [comment]
It's normal as far as I know - this happens because of mismatch between the image size and the media size, or because of the Chrome OS partition layout. Not sure really which explanation is the correct one, but things work well despite of this. Make sure the image looks sane - the kernel partition should not start with zero bytes.

**Comment by dimkr** [comment]
And torrents sounds like a great idea - something I intend to do in the future unless someone beats me to it.

**Comment by dimkr** [comment]
Forgot to mention: you'll have to disable kernel signature verification, in case you haven't. Maybe I should document this, plus Libreboot installation.

As far as I know Devsus generated images should be bootable with the stock firmware and today I tested an image against an old Coreboot image from 2015 and the latest stable Libreboot release, just to be sure.

**Comment by nikolas-n** [comment]
I have already installed Libreboot and disabled kernel signature verification. In fact, I have the Arch Linux ARM distro running from the SD card on the machine. When I try to boot the image as it is, i.e. without having repaired the partitions, I get a beep and I'm returned back to the Libreboot screen.

I have created a torrent for each image but I'm not sure the two images are compiled right, so I don't think is a good idea to share it (or is it?). Would it be possible that you upload it somewhere (for a short period of time maybe), so that we could download it?

Anyway, I'll check again the partitions in case they're not as the machine expects them to be.
